--- LedBlinker/Main.cpp
+++ LedBlinker/Main.cpp
@@ -10,2 +10,3 @@
 #include <LedBlinker/Top/LedBlinkerTopology.hpp>
 #include <Fw/Logger/Logger.hpp>
+#include <zephyr/sys/printk.h>
@@ -29,2 +30,3 @@
 {
+    printk("Starting main\n");
     // Object for communicating state to the reference topology
@@ -33,3 +35,5 @@
     // Setup topology
+    printk("Setting up topology...\n");
     LedBlinker::setupTopology(inputs);
+    printk("Topology setup complete!\n");
     
